src/config/media/pipewire.md

```diff
@@ -45,96 +44,124 @@ Install the `wireplumber` package to use this session manager with PipeWire.
 > prior override was to disable `pipewire-media-session`, deleting the custom
 > configuration may be sufficient.
 
-There are several methods for starting `wireplumber` alongside `pipewire`. If
-your window manager or desktop environment auto-start mechanism is used to start
-`pipewire`, it is recommended to use the same mechanism for starting
-`wireplumber`. The `wireplumber` package includes a `wireplumber.desktop`
-Desktop Entry file that may be useful in this situation.
-
-> Be aware that `wireplumber` must launch *after* the `pipewire` executable. The
-> Desktop Application Autostart Specification makes no provision for ordering of
-> services started via Desktop Entry files. When relying on these files to
-> launch `pipewire` and `wireplumber`, consult the documentation for your window
-> manager or desktop environment to determine whether proper ordering of
-> services can be achieved.
-
-If proper ordering of separate `pipewire` and `wireplumber` services is
-infeasible, it is possible to configure `pipewire` to launch the session manager
-directly. This can be accomplished by running
+Currently, there is only one session manager available: WirePlumber. To
+configure PipeWire to use this session manager and ensure proper startup
+ordering, PipeWire should be configured to launch the session manager directly.
+This can be accomplished by running
 
 ```
 # mkdir -p /etc/pipewire/pipewire.conf.d
 # ln -s /usr/share/examples/wireplumber/10-wireplumber.conf /etc/pipewire/pipewire.conf.d/
 ```
 
-for system configurations or, for per-user configurations, running
+for system-wide configuration, or
 
 ```
 $ true "${XDG_CONFIG_HOME:=${HOME}/.config}"
```

Comment:
Instead of `true` I'm used to using the `:` command.

Reply:
@ahesford suggested using true because it's harder to miss than `:` (this came from the last rewrite)